build/get_new_operator_sdk.sh

```diff
@@ -3,10 +3,11 @@
 REL=$(dirname "$0")
 ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
 OS=$(uname | awk '{print tolower($0)}')
-OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.5.0
+VERSION="$1"
```

Comment:
You can set a default if you want here so that you can use the script as you were previously. This keeps it at 1.5.0 as your default, but at least it can be reused this way too.

```suggestion
VERSION="${1:-1.5.0}"
```